src/xia2/Wrappers/Dials/Spotfinder.py

```diff
@@ -88,6 +89,9 @@ def set_filter_ice_rings(self, filter_ice_rings):
         def set_gain(self, gain):
             self._gain = gain
 
+        def set_maximum_trusted_value(self, maximum_trusted_value):
+            self._maximum_trusted_value = maximum_trusted_value
+
```

Comment:
If this attribute is essentially a simple parameter, why do we bother with the setter at all?  Why not simply make the attribute public (i.e. `self.maximum_trusted_value`) and assign to it, rather than calling a method to do the same thing?

Reply:
Your comment is reasonable and valid - however the "house style" is the getter / setter pattern & this is fitting in with that. 

The question of "is the house style wrong" would be a reasonable follow up.